### PR TITLE
Added Zero routing to QBS

### PR DESCRIPTION
**Pull request by jonnyshaw89**
### What is the context of this PR?
Added routing on Zero to QBS to skip breakdown questions

### CODE REVIEW

**Review by dwyeradam — CHANGES_REQUESTED** [review]
Routing doesn't work. If I select "No I need to change this" then I still go to the summary.

**Review by dwyeradam — APPROVED** [review]

**Comment by shahi645** [comment]
@jonnyshaw89 needs a yarn format

**Comment by jonnyshaw89** [comment]
@shahi645 already done

File: data/en/2_0001.json [hunk 1]
```diff
@@ -98,6 +104,54 @@
...
+                                {
+                                    "label": "No I need to change this",
+                                    "value": "No"
+                                }
+                            ],
+                            "mandatory": true
+                        }],
+                        "id": "confirm-zero-employees-question",
+                        "title": "On {{exercise.start_date|format_date}}, the number of employees for {{respondent.trad_as_or_ru_name}} was <em>0</em>, is this correct?"
+                    }],
```
Comment (shahi645) [inline data/en/2_0001.json:137]: on the spec this `,` is within the <em> tags, but might want to check that with @charlotte-hirst

**Review by shahi645 — APPROVED** [review]

**Pull request merged**